assets.sh

```diff
@@ -51,7 +51,6 @@ KUBERNETES_ASSETS=(
 PIT_IMAGE_ID=5.2.2
 PIT_ASSETS=(
     "https://artifactory.algol60.net/artifactory/csm-images/stable/pre-install-toolkit/${PIT_IMAGE_ID}/pre-install-toolkit-${PIT_IMAGE_ID}-${NCN_ARCH}.iso"
-    "https://artifactory.algol60.net/artifactory/csm-images/stable/pre-install-toolkit/${PIT_IMAGE_ID}/installed.deps-${PIT_IMAGE_ID}-${NCN_ARCH}.packages"
```

Comment:
This URL existing causes the file to exist in the tarball.
I don't think this file actually needs to be here since `hack/embedded-repo.sh` downloads the file again anyway.

Reply:
Sounds good